### ISSUE TITLE
True Polymorph Description Incorrect

### ISSUE THREAD

**Issue by MRiddickW** [Traneptora/grimoire#112]
The last sentence of the first paragraph of the Grimoire states “If you concentrate on this spell for the full duration, the transformation becomes permanent.”

In the SRD, it states “If you concentrate on this spell for the full duration, the transformation lasts until it is dispelled.”

**Comment by Traneptora** [comment]
The Players' Handbook in this case disagrees with the System Reference Document, and my policy is to use PHB instead of SRD wording if there's a conflict. I also checked the PHB errata and this has not been changed.

**Comment by MRiddickW** [comment]
That's fair. After I posted this, I did some for more research and it seems that a certain PDF that may be floating around was actually some sort of prerelease copy or something, thus why it never showed up in the public errata. If you don't mind, double check a physical version to see if yours jives with mine. I'm not sure which printing mine is, I just know it's not the first. Here's a picture of the relevant text my book.

![IMG_E1488](https://user-images.githubusercontent.com/45296126/69027310-1deb6f80-0994-11ea-895c-24343f2b031f.JPG)
